docs/discussion/maintenance-schedules.rst

```diff
@@ -0,0 +1,78 @@
+Maintenance Schedules
+===========================================================
+
+This is a record of the intended maintenance schedules for various releases
+and dependency support.
+
+Python Version Support
+-----------------------------------------------------------
+
+PursuedPyBear projects will support at least the latest version of
+`pypy <https://www.pypy.org/download.html>`_ and the latest two versions of
+`C Python <https://www.python.org/downloads/>`_
+
+We tend to start testing new versions of cPython when they reach the release
+candidate stage, but don't guarantee compatibility until at least one ppb
+release after the latest Python release.
+
+We drop older versions of Python when we begin using newer features. This
+means the oldest version of Python ppb will support may be older than the
```

Comment:
this is hard to understand.

Reply:
I'm not totally sure how best to improve it.

Like, I'm basically wanting to make it clear the guarantee is "at least" but not "only" the last two.

Reply:
Reworded, added an example.